src/packagedcode/jar_manifest.py

```diff
@@ -278,7 +248,7 @@ def dget(s):
     package['homepage_url'] = dget('Implementation-URL') or dget('Implementation-Url')
 
     # Bundle-DocURL: http://logging.apache.org/log4j/1.2
-    package['documentation_url'] = dget('Bundle-DocURL')
+    doc_url = dget('Bundle-DocURL')
```

Comment:
I get what you are doing, but we are losing data here.

Reply:
We are returning this `package` as-is to be used for creating a PackageData class, see https://github.com/nexB/scancode-toolkit/blob/update-maven-detections/src/packagedcode/maven.py#L138 with `models.PackageData(**manifest,)`. We don't have a `documentation_url` or a `notes` field in PackageData, and this is why the Manifest parsing used to fail and not return anything. I'm just ensuring we return whatever we can, instead of crashing and not returning anything. And since I'm having this in `extra_data`, we're not really losing any data IMHO